### PR TITLE
Created the change log document

### PR DESCRIPTION
**Pull request by mariari**
Creates the changelog file, and adds the meaning to the contributors guidleine.

### CODE REVIEW

File: doc/CONTRIBUTING.md [hunk 1]
```diff
@@ -40,6 +40,27 @@ The mainline branch is `develop` (this will change post-1.0.0). Feature developm
 Name your feature branches with a name or handle of the branch owner as a prefix, followed by a brief description preceded by a slash, e.g. `cwgoes/eal-inference-bugfix`.
 
 Pushing minor changes (typo fixes) to another person's branch is fine. Ask before pushing major changes.
+## Version changelogs
+
+The format for the changelog follows the PVP format below.
+
+```
+  ︻ ︻ ︻ ︻
```
Comment (cwgoes) [inline doc/CONTRIBUTING.md:48]: I find this diagram pretty confusing, could we write it differently perhaps? Just list the sub-version-numbers.

File: doc/CONTRIBUTING.md [hunk 2]
```diff
@@ -40,6 +40,21 @@ The mainline branch is `develop` (this will change post-1.0.0). Feature developm
...
+The format for the changelog follows the PVP format below.
+
+V A.B.C.D
+
+A and B are both major versions. C is the Minor version, and D is the patch, in other words it looks like
+
+V Major.Major.Minor.Patch
+
+Major release will bump the major version, milestones bump the minor
+version, and pull requests bump the patch number.
```
Comment (Acentelles) [inline doc/CONTRIBUTING.md:54]: Do you have a reference about this? I don't think pull requests have anything to do with patch, minor or major version. E.g. if a PR brings breaking changes, I think it should update the changelog file with a major bump

File: doc/CONTRIBUTING.md [hunk 3]
```diff
@@ -40,6 +40,21 @@ The mainline branch is `develop` (this will change post-1.0.0). Feature developm
...
 Pushing minor changes (typo fixes) to another person's branch is fine. Ask before pushing major changes.
+## Version changelogs
+
+The format for the changelog follows the PVP format below.
+
+V A.B.C.D
+
+A and B are both major versions. C is the Minor version, and D is the patch, in other words it looks like
+
+V Major.Major.Minor.Patch
```
Comment (Acentelles) [inline doc/CONTRIBUTING.md:51]: Is the V necessary?

File: doc/CONTRIBUTING.md (hunk 2 above)
Comment (mariari) [reply doc/CONTRIBUTING.md:54]: Fair, I think I worded it poorly, I guess I should specify "a pull request should bump one of these numbers)

File: doc/CONTRIBUTING.md (hunk 3 above)
Comment (mariari) [reply doc/CONTRIBUTING.md:51]: No I'll get rid of it now

**Review by Acentelles — APPROVED** [review]

**Pull request merged**